.github/workflows/build-and-test.yml

```diff
@@ -75,6 +75,9 @@ jobs:
     runs-on: ubuntu-latest
     needs: [setup-environment]
     steps:
+      - run: sudo apt-get -qq update
+      - name: Install libsystemd-dev
+        run: sudo apt-get install libsystemd-dev
```

Comment:
For my benefit, will you please explain why this is needed?

Reply:
On Linux, Promtail requires the systemd headers to be installed for Journal support.
Promtail brought package [sdjournal](https://github.com/coreos/go-systemd/blob/main/sdjournal/journal.go) that uses C.
To make compilation work we need to install libsystemd-dev

Reply:
Thanks for explaining